packages/flutter/lib/src/rendering/semantics.dart

```diff
@@ -126,24 +161,31 @@ class SemanticsNode extends AbstractNode {
 
   bool get _shouldMergeAllDescendantsIntoThisNode => mergeAllDescendantsIntoThisNode || _inheritedMergeAllDescendantsIntoThisNode;
 
+  /// Whether this node responds to tap gestures.
   bool get canBeTapped => _flags[_SemanticFlags.canBeTapped];
   set canBeTapped(bool value) => _setFlag(_SemanticFlags.canBeTapped, value, needsHandler: true);
 
+  /// Whether this node responds to long-press gestures.
   bool get canBeLongPressed => _flags[_SemanticFlags.canBeLongPressed];
   set canBeLongPressed(bool value) => _setFlag(_SemanticFlags.canBeLongPressed, value, needsHandler: true);
 
+  /// Whether this node responds to horizontal scrolling.
   bool get canBeScrolledHorizontally => _flags[_SemanticFlags.canBeScrolledHorizontally];
   set canBeScrolledHorizontally(bool value) => _setFlag(_SemanticFlags.canBeScrolledHorizontally, value, needsHandler: true);
 
+  /// Whether this node responds to vertical scrolling.
   bool get canBeScrolledVertically => _flags[_SemanticFlags.canBeScrolledVertically];
   set canBeScrolledVertically(bool value) => _setFlag(_SemanticFlags.canBeScrolledVertically, value, needsHandler: true);
 
+  /// Whether this node has Boolean state that can be toggled on and off.
```

Comment:
specifically, whether it has a boolean state that can in principle be changed by user interaction.

As opposed to the boolean state "is enabled", for instance.